x-pack/plugins/observability/public/pages/rules/index.tsx

```diff
@@ -171,6 +172,23 @@ export function RulesPage() {
         'data-test-subj': 'rulesTableCell-name',
         render: (name: string, rule: RuleTableItem) => <Name name={name} rule={rule} />,
       },
+      {
+        field: 'tags',
+        name: '',
+        sortable: false,
+        width: '50px',
+        'data-test-subj': 'rulesTableCell-tagsPopover',
+        render: (tags: string[], item: RuleTableItem) => {
+          return tags.length > 0
+            ? triggersActionsUi.getRuleTagBadge({
+                isOpen: tagPopoverOpenIndex === item.index,
+                tags,
+                onClick: () => setTagPopoverOpenIndex(item.index),
+                onClose: () => setTagPopoverOpenIndex(-1),
+              })
+            : null;
```

Comment:
Rendering `null` will not throw an error?

Reply:
I used similar code with the Stack Management rules list page https://github.com/elastic/kibana/pull/130721/files#diff-0b08d93b140ecb5d9343573bcb0c0fc3b18ca62347555259b247c9cf36b02e5cL627